### PR TITLE
vim-patch:8.1.{972,974}

### PR DESCRIPTION
**Pull request by butwerenotthereyet**
Add the `<C-W>gt` and `<C-W>gT` aliases for `gt` and `gT` respectively.  

These were added for vim's `:terminal` but apply in normal mode and are [already documented](https://github.com/neovim/neovim/commit/41fe644124140c4a303de4a5a84e28efec918274) to be normal commands in neovim.

### CODE REVIEW

File: test/functional/ui/tabpage_spec.lua [hunk 1]
```diff
@@ -0,0 +1,38 @@
+local helpers = require('test.functional.helpers')(after_each)
```
Comment (justinmk) [inline test/functional/ui/tabpage_spec.lua]: let's put this in `test/functional/normal/tabpage_spec.lua`. The /ui/ subdir is for actual UI tests, i.e. `screen:expect()` things. 

(Though /normal/ isn't really a great place for this either, but that's why I don't like all of these categories...)

Comment (butwerenotthereyet) [reply test/functional/ui/tabpage_spec.lua]: > let's put this in test/functional/normal/tabpage_spec.lua.

Fixed.

> The /ui/ subdir is for actual UI tests, i.e. screen:expect() things.

Noted!

File: test/functional/normal/tabpage_spec.lua [hunk 2]
```diff
@@ -0,0 +1,38 @@
+local helpers = require('test.functional.helpers')(after_each)
+
+local clear = helpers.clear
+local command = helpers.command
+local eq = helpers.eq
+local feed = helpers.feed
+local eval = helpers.eval
+
+describe('ui/tabpage', function()
```
Comment (justinmk) [inline test/functional/normal/tabpage_spec.lua]: - can remove `ui/` prefix here since it's not a UI test
- file should have 2-space indent

Comment (butwerenotthereyet) [reply test/functional/normal/tabpage_spec.lua]: Fixed.

File: test/functional/normal/tabpage_spec.lua [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+local clear = helpers.clear
+local command = helpers.command
+local eq = helpers.eq
+local feed = helpers.feed
+local eval = helpers.eval
+
+describe('tabpage', function()
+  before_each(clear)
+
+  it('advances to the next page via <C-W>gt', function()
```
Comment (janlazo) [inline test/functional/normal/tabpage_spec.lua:12]: This test is not running on a terminal buffer unlike the test in `test_terminal.vim`. Same issue with the other test in this file.

**Pull request merged**